news/2018-03-27-the-qat-gazette-march-2018.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "The QAT Gazette: March 2018"
+date: 2018-03-27 09:00:00 +0000
+---
+Restructuring of the Quality Assurance Team, more communication and transparency regarding decision-making, and a greater focus on actually ensuring quality in the Qualified section again... Big changes to discuss this month, so read inside to find out all the details!
+
+![](/wiki/shared/news/banners/qat-blog-logo.png)
+
+## Opening 
+by [JBHyperion](https://osu.ppy.sh/users/4879508)
+
+Welcome back everyone! JBHyperion here with another month's worth of news and developments!
```

Comment:
next month's?

Reply:
We're reviewing what happened this month, rather than previewing what is going to happen next month